doc/source/devel/governance.rst

```diff
@@ -40,7 +40,7 @@ project in concrete ways, such as:
 among other possibilities. By contributing to the project, community members
 can directly help to shape its future.
 
-Contributors should read the :ref:`contributor_guide` and our :ref:`code_of_conduct`.
+Contributors should read the :ref:`chap_devguide` and our :ref:`code_of_conduct`.
```

Comment:
Do we have a `code_of_conduct` ref?   As you know, I personally prefer 'community guidelines' as a document title.

Reply:
I think you added these in for aliases:

https://github.com/nipy/nibabel/blob/7fc471f8895188497a75914fb137f9a048916128/doc/source/devel/devguide.rst#L114-L121

Can `sed` the refs anyway...